src/fhir/resource_helpers.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Coding,
+  Identifier,
+  Narrative,
+  ObservationComponent,
+  Reference
+} from "@smile-cdr/fhirts/dist/FHIR-R4/classes/models-r4";
+
+export const reference = (refType: string, id: string): Reference => {
+  return {reference: `${refType}/${id}`, type: refType}
+};
+
+export const makeGoshAssigner = (valueType: string) => {
+  return {use: Identifier.UseEnum.Official, assigner: {display: `GOSH ${valueType}`}}
+}
+
+export function generatedNarrative(...parts: string[]) {
+  return {status: Narrative.StatusEnum.Generated, div: `${parts.join(" ")} from FHIR genomics app`};
+}
```

Comment:
Can we guarantee we will always be passing strings as arguments into this function?

Reply:
mmm interesting question, I think so - but can definitely reassess if we come across a case that breaks this assumption